@kiva/kv-components/src/vue/KvTooltip.vue

```diff
@@ -109,6 +122,27 @@ export default {
 			},
 		}));
 
+		const popperRef = ref(null);
+
+		const getControllerElement = () => {
+			if (typeof props.controller === 'string') {
+				return document.getElementById(props.controller);
+			}
+			return props.controller;
+		};
+
+		const handleKvPopperVisibility = (isShowing) => {
+			emit('tool-tip-visible', isShowing);
+		};
+
+		const triggerHover = (enter = true) => {
+			const element = getControllerElement();
+			if (element) {
+				const eventType = enter ? 'mouseover' : 'mouseout';
```

Comment:
Noting that I initially attempted to call methods through the ref on KvPopper but couldn't get anything to trigger or response. Thus I opted for simulating mouse events that were already initiated and handled by the KvPopper component.